pkg/controllers/podannotator/common.go

```diff
@@ -19,11 +21,13 @@ func updatePods(ns v1.Namespace, cfg platformv1.PodMutaterConfig, pods ...v1.Pod
 
 		podChanged := false
 
-		for k, v := range ns.Annotations {
-			if _, f := cfg.AnnotationsMap[k]; f { // if annotation is whitelisted
-				if _, podHasAnnotation := pod.Annotations[k]; !podHasAnnotation { // if pod already has annotation, don't inherit
-					pod.Annotations[k] = v
-					podChanged = true
+		for k1, v1 := range ns.Annotations {
```

Comment:
Can we try to make this snippet a little more readable with better variables names  and/or helper functions?